scripts/transact/migrateTunnelLandOnL1.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs-extra';
```

Comment:
add a small doc on how to use the script

Reply:
Added comments for that.